.lintstagedrc.json

```diff
@@ -0,0 +1,14 @@
+{
+    "*.css": [
+      "10up-scripts lint-style"
```

Comment:
can you update this to `10up-toolkit` as per https://github.com/10up/wp-scaffold/pull/35